### PR TITLE
MV3 feature release clarification

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Updates support information for Manifest V3 features which were in preview prior to Firefox 109.

#### Related issues

Related changes to content made in https://github.com/mdn/content/pull/24916

### CODE REVIEW

**Review by queengooborg — CHANGES_REQUESTED** [review]
Notes are actually formatted as HTML rather than Markdown, so we'll need to use `<a href=""></a>` rather than `[]()`.

**Review by Rob--W — CHANGES_REQUESTED** [review]
Github does not offer the ability to comment on non-changed lines, so here are more pieces of feedback:

- Drop mentions of "MV3 preview" from the BCD. The pre-release testing phase is not relevant to devs, especially because MV3 support has been enabled by default in Firefox 109+.
- manifest/action.js: should refer to the action MDN articles instead of the browser_action one.
  - Not strictly part of this PR, but it would also be nice tot interlink each other (action / browser_action at "See also" in mdn/content )
- manifest/content_security_policy.json: Like Firefox, Chrome has also dropped the object-src requirement, since version 111 - https://bugs.chromium.org/p/chromium/issues/detail?id=1320785#c10

File: webextensions/api/scripting.json [hunk 1]
```diff
@@ -2,6 +2,25 @@
   "webextensions": {
     "api": {
       "scripting": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/",
+          "support": {
+            "chrome": {
+              "version_added": "88",
+              "notes": "Available for use in Manifest V3 or later."
```
Comment (rebloor) [inline webextensions/api/scripting.json:10]: @Rob--W should I repeat this against all the sub features?

File: webextensions/api/scripting.json
Comment (rebloor) [inline webextensions/api/scripting.json:1]: @Rob--W do you think we need the "Available for use in Manifest V2 or later." notes against Safari support?

File: webextensions/manifest/action.json [hunk 2]
```diff
@@ -3,25 +3,22 @@
...
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action",
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action",
           "support": {
             "chrome": {
               "version_added": "88",
               "notes": [
                 "Available for use in Manifest V3 or later.",
-                "If an extension defines a browser action, it is not allowed to define a page action as well."
+                "If an extension defines an action, it is not allowed to define a page action as well."
```
Comment (rebloor) [inline webextensions/manifest/action.json:12]: @Rob--W correct?

**Comment by rebloor** [comment]
> Not strictly part of this PR, but it would also be nice tot interlink each other (action / browser_action at "See also" in mdn/content )

It not the "This key replaces [browser_action](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action) in Manifest V3 extensions." and the reciprocal version in browser_action adequate?

File: webextensions/manifest/action.json (hunk 2 above)
Comment (Rob--W) [reply webextensions/manifest/action.json:12]: Chrome does not support `page_action` in MV3 at all. So "not allowed to define a page action" is not conditional on the action key being present. In MV2, Chrome does not allow browser_action and page_action to co-exist (and doesn't support the action key either).

FYI: Safari does not support page_action nor browser_action in MV3 either (only `action`). In MV2 Safari does support page_action and browser_action.

(and for completeness: Firefox supports browser_action only in MV2, action only in MV3, and page_action in MV2+MV3).

Comment (rebloor) [reply webextensions/manifest/action.json:12]: Thanks @Rob--W I've removed this note.

File: webextensions/manifest/background.json [hunk 3]
```diff
@@ -64,6 +64,7 @@
               "firefox": {
                 "version_added": "48",
                 "notes": [
+                  "From Firefox 109, persistent and non-persistent pages are fully supported for Manifest V2 and V3.",
```
Comment (Rob--W) [inline webextensions/manifest/background.json:67]: MV3 only supports non-persistent background pages. It does not support the `persistent` key. Drop this line.

Also, as a cleanup, drop the two notes below about the preferences. Now that the feature has officially shipped, the prefs in older browser versions are not relevant.

**Review by Rob--W — APPROVED** [review]
Current changes here look good.

To complete the full scope of "MV3 feature clarification", please also update:
- https://github.com/mdn/browser-compat-data/blob/22e0f5f1707f29f0c646ea89f5ba5ea7734fa48a/webextensions/manifest/web_accessible_resources.json
  - Anything with a property that is supported in Firefox should have its version changed to 109 (MV3), because the object format (in an array) of `web_accessible_resources` is MV3-only. In MV2, web_accessible_resources was merely an array of strings. This implies that properties are only supported with the release of MV3.


That change is trivial, so feel free to merge after making the change (or submit a follow-up PR if you wish).

**Comment by rebloor** [comment]
Thanks @Rob--W, I have made the final change you requested and merged.

**Pull request merged**